pyproject.toml

```diff
@@ -71,6 +71,7 @@ test = [
     "python-jose[cryptography] >=3.3.0,<4.0.0",
     "pyyaml >=5.3.1,<7.0.0",
     "passlib[bcrypt] >=1.7.2,<2.0.0",
+    "trio >=0.19,<0.22.0",
```

Comment:
Was pinning `trio` to a version older than the latest one intentional?

Reply:
the `trio` module is declared for the _test dependencies_. It should not be a big deal

Reply:
I don't remember exactly. IIRC it was a fix for the fails in tests related to the trio package. It was not related to the update of the starlette version.

https://github.com/tiangolo/fastapi/pull/5547 probably fixed it.

I'll drop this change.

Reply:
Thank you for looking into it!